bin/compute_gene_heatmap.R

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env Rscript
```

Comment:
no license / author for the script?

Reply:
Its us, we will submit the paper immediately after this is released and after all its just part of the pipeline :)